site/docs/configuration/expected-outputs/model-graded/context-faithfulness.md

```diff
@@ -34,18 +34,28 @@ tests:
         threshold: 0.8
 ```
 
-### Extracting from provider responses
+### Dynamically from the provider's response
 
-If your provider returns context within the response, use `contextTransform`:
+If your provider returns context within the response, use `contextTransform` to extract it. For example:
+
+```typescript
+interface Output {
+  content: string;
+  citations: string;
+  retrieved_docs: {
+    content: string;
+  }[];
+}
+```
 
 ```yaml
 assert:
   - type: context-faithfulness
-    contextTransform: 'output.context'
+    contextTransform: 'output.citations'
```

Comment:
add a comment explaining this is a javascript function (output) => { return .... } where output can be an object or string